### ISSUE TITLE
[A11Y] Focus ring missing from text input

### ISSUE THREAD

**Issue by tomayac** [o-t-w/copypaste-webcomponent#2]
The text input has no visible focus ring. This seems to be caused by this line of CSS: https://github.com/o-t-w/copypaste-webcomponent/blob/523570ab65b974a02bbaaa2bfa9976942c3ecaa6/index.js#L76
